extensions/logging-json/deployment/src/main/java/io/quarkus/logging/json/deployment/LoggingJsonSteps.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.logging.json.deployment;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.LogConsoleFormatBuildItem;
+import io.quarkus.logging.json.runtime.JsonConfig;
+import io.quarkus.logging.json.runtime.LoggingJsonRecorder;
+
+public final class LoggingJsonSteps {
```

Comment:
Maybe call it `*Processor`? It's the generally used name. There are a few different ones but let's try to settle on one convention.

Reply:
I never liked `Processor` because it doesn't really process anything; these classes are just collections of disparate (possibly even only casually related) build steps.  So using the convention of `Steps` makes the most sense as a convention in my mind.